cernopendata_client/opendata_analysis_query.py

```diff
@@ -1,208 +0,0 @@
-from __future__ import print_function
```

Comment:
Are we ready to delete this file already?  If so, you can close issue #20 as well.

Reply:
Yes, I think so. I have moved the used two functions in search.py and other functions are not used anywhere.